From 6c956665947998cbb4f39373a91e991c072f1794 Mon Sep 17 00:00:00 2001
From: Danny Milosavljevic <dannym@friendly-machines.com>
Date: Tue, 24 Mar 2026 00:45:32 +0000
Subject: [PATCH] client/wayland: Fix typo "waylayd" -> "wayland" in
 Makefile.am

The misspelled variable name caused the METADATADIRS and VAPIDIRS
settings to be silently ignored by automake.

BUG=https://github.com/ibus/ibus/pull/2881
---
 client/wayland/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/client/wayland/Makefile.am b/client/wayland/Makefile.am
index 25269557f..3275d1c5a 100644
--- a/client/wayland/Makefile.am
+++ b/client/wayland/Makefile.am
@@ -184,8 +184,8 @@ MAINTAINERCLEANFILES += $(VAPIGEN_VAPIS)
 DISTCLEANFILES += $(VAPIGEN_VAPIS)
 $(VAPIGEN_VAPIS): $(INTROSPECTION_GIRS)
 ibus_wayland_im_1_0_vapi_DEPS = glib-2.0 gobject-2.0 gio-2.0 ibus-1.0
-ibus_waylayd_im_1_0_vapi_METADATADIRS = $(srcdir) $(top_srcdir)/bindings/vala
-#ibus_waylayd_im_1_0_vapi_VAPIDIRS = $(top_srcdir)/bindings/vala
+ibus_wayland_im_1_0_vapi_METADATADIRS = $(srcdir) $(top_srcdir)/bindings/vala
+#ibus_wayland_im_1_0_vapi_VAPIDIRS = $(top_srcdir)/bindings/vala
 ibus_wayland_im_1_0_vapi_FILES = \
     $(INTROSPECTION_GIRS) \
     $(srcdir)/ibus-wayland-im-custom.vala \
